core/server/api/v2/settings.js

```diff
@@ -177,7 +177,7 @@ module.exports = {
         headers: {
             disposition: {
                 type: 'yaml',
-                value: 'Attachment; filename="routes.yaml"'
+                value: 'routes.yaml'
```

Comment:
Left the extension part to the controller, as there's no clear reason to extract it to the header handler. Maybe you have another perspective on it @kirrg001 ?